roles/install-app/tasks/main.yml

```diff
@@ -101,3 +110,26 @@
     - install
     - app
     - docs
+
+#TODO use ansible_selinux
+- name: Permit nginx to connect to backend
+  when: ansible_distribution == "CentOS"
+  seboolean:
+    name: httpd_can_network_connect
+    state: yes
+    persistent: yes
+  tags:
+    - install
+    - app
+
+# the optionnal yaml.Cloader is never installed by pip
+# This hack install it in virtualenv
```

Comment:
Why is that it is never installed on CentOS?

I'm not sure on CentOS, but pip might not be installing the `yaml.Cloader` because you are installing PyYAML as a system wide package, and maybe pip is not overriding it in the virtual environment.

As for Debian, you might need the `libyaml-dev` package in order for pip to produce the `yaml.Cloader` one, since it needs to be compiled.